test/functional/test-document-info.js

```diff
@@ -60,9 +61,7 @@ describe('document-info', () => {
 
   it('should provide the pageViewId', () => {
     return getWin('https://twitter.com/').then(win => {
-      win.Math.random = () => {
-        return 0.123456789;
-      };
+      sinon./*OK*/stub(win.Math, 'random', () => 0.123456789);
```

Comment:
I didn't want to create a sandbox just to stub something that didn't need to be un-stubbed (this is an iframe window). I guess I could stop being lazy....